tests/e2e/helpers.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	//nolint:golint,stylecheck
+	. "github.com/onsi/ginkgo"
+
+	//nolint:golint,stylecheck
+	. "github.com/onsi/gomega"
+)
+
+var (
+	selinuxdInAContainer  bool
+	selinuxdContainerName string
+)
+
+const (
+	// timeout for selinuxd to report it's ready
+	selinuxdReadyTimeout float64 = 120
```

Comment:
Let's see if this is enough, I've seen upwards of 3 minutes in my k8s environment..

Reply:
done.